circuit_knitting/cutting/cutting_evaluation.py

```diff
@@ -306,13 +336,22 @@ def _generate_cutting_experiments(
             decomp_qc = decompose_qpd_instructions(
                 subcircuit, subcirc_qpd_gate_ids[i], map_ids_tmp
             )
-            subexperiments[-1].append([])
+            subexperiments_legacy[-1].append([])
             so = subsystem_observables[label]
             for j, cog in enumerate(so.groups):
                 meas_qc = _append_measurement_circuit(decomp_qc, cog)
-                subexperiments[-1][-1].append(meas_qc)
+                subexperiments_legacy[-1][-1].append(meas_qc)
+
+    # If the input was a single quantum circuit, return the subexperiments as a list
+    subexperiments_out: list[QuantumCircuit] | dict[
+        str | int, list[QuantumCircuit]
+    ] = subexperiments_dict
```

Comment:
`subexperiments_dict` is really a `defaultdict`.  Let's convert it to an actual `dict` before we return it.

```suggestion
    ] = dict(subexperiments_dict)
```